activejob/lib/active_job/enqueuing.rb

```diff
@@ -9,6 +9,32 @@ module ActiveJob
   # why the adapter was unexpectedly unable to enqueue a job.
   class EnqueueError < StandardError; end
 
+  class << self
+    # Push many jobs onto the queue at once without running enqueue callbacks.
+    # Returns the number of jobs successfully enqueued. If a queue adapter does not
+    # implement enqueue_all, its jobs are enqueued individually.
+    def perform_all_later(jobs)
```

Comment:
Do you think it might be worth accepting either splats or an array?

I'm thinking for small cases splat may be more natural, e.g. `perform_all_later(Job.new(1), Job.new(2))`

Reply:
That makes sense to me. I'll add that.